app/src/main/java/com/nervousfish/nervousfish/cryptography/KeyGenerator.java

```diff
@@ -0,0 +1,62 @@
+package com.nervousfish.nervousfish.cryptography;
+
+import java.security.KeyFactory;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.RSAPrivateKeySpec;
+import java.security.spec.RSAPublicKeySpec;
+
+public class KeyGenerator {
```

Comment:
@jverbraeken do we require class JavaDoc for classes, if so we should add a Checkstyle rule for that and add it here...